api/v1beta1/disruption_webhook.go

```diff
@@ -46,6 +46,7 @@ var (
 	defaultNamespaceThreshold     float64
 	defaultClusterThreshold       float64
 	handlerEnabled                bool
+	maxDuration                   time.Duration
```

Comment:
this parameter should also be able to not be defined I think (in case of the chaos-controller being in staging env)

Reply:
It's already the case, you don't need to define the `maxDuration` in the configuration and it will not be interpreted.

Reply:
didn't know, nice 👍

Reply:
Your point is true, I debug it and if the maxDuration is not defined it has the 3600 default value. So I need to change that 👍